main.py

```diff
@@ -53,6 +53,10 @@ def init(config:Config):
 
     logging.info(f"Starting Chiadog ({version()})")
 
+    # Remove the Pygtail offset file if it exists
+    # Fixes a bug where hard reboot corrupts the offset file
+    Config.get_offset().unlink(missing_ok=True)
```

Comment:
Nitpick: I'd rename the `get_offset` function to `get_log_offset_path`. It'd be a bit more descriptive and clear from the function call what happens here.